igvm/settings.py

```diff
@@ -33,7 +33,7 @@
 # Mapping to determine the libvirt CPU model based on serveradmin hw_model
 KVM_HWMODEL_TO_CPUMODEL = {
     'Nehalem': ['Dell_M610', 'Dell_M710'],
-    'SandyBridge': ['Dell_M620', 'Dell_M630', 'Dell_R620'],
+    'SandyBridge': ['Dell_M620', 'Dell_M630', 'Dell_M640', 'Dell_R620'],
```

Comment:
This is not sandybridge though, right?

Reply:
This is intended so that we can do live migrations between those systems.

Reply:
Hu, but we still also have M610s. Does that mean we can not migrate between those and the others either?